assets/sass/components/analytics-4/audience-segmentation/_googlesitekit-audience-segmentation-setup-cta.scss

```
@@ -0,0 +1,86 @@
+/**
+ * Audience Segmentation Setup CTA styles.
+ *
+ * Site Kit by Google, Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.googlesitekit-plugin {
+	.googlesitekit-audience-segmentation-setup-cta-widget {
+		.googlesitekit-widget-audience-segmentation-primary-cell {
+			padding: 0 24px 24px;
+
+			@media (max-width: $bp-mobileOnly) {
+				padding-bottom: 0;
+			}
+		}
+
+		.googlesitekit-widget-audience-segmentation-text__wrapper {
+			.googlesitekit-publisher-win__title {
+				@media (max-width: $bp-mobileOnly) {
+					font-weight: 500;
+					margin-bottom: 10px;
+				}
+
+				@media (min-width: $bp-tablet) {
+					margin-bottom: 8px;
+				}
+
+				@media (min-width: $bp-desktop) {
+					margin-bottom: 25px;
+				}
+			}
+
+			p {
+				@media (min-width: $bp-tablet) {
+					margin-top: 0;
+				}
+
+				@media (max-width: $bp-mobileOnly) {
+					font-size: 12px;
+					margin-top: 0;
+				}
+			}
+		}
+
+		.googlesitekit-widget-audience-segmentation-actions__wrapper {
+			align-items: center;
+			column-gap: 30px;
+			display: flex;
+			flex-wrap: wrap;
+			margin: 25px 0;
+			row-gap: 24px;
+
+			@media (min-width: $bp-desktop) {
+				margin: 25px 0 0;
+			}
+
+			@media (max-width: $bp-mobileOnly) {
+				column-gap: 10px;
+			}
+
+			a,
+			button {
+				@media (max-width: $bp-mobileOnly) {
+					font-weight: 500;
+				}
+			}
+		}
+	}
+
+	.googlesitekit-widget-audience-segmentation-svg__wrapper {
+		bottom: -5px;
+		position: relative;
```

@10upsimon I've been borrowing from this code for the Consent Mode setup banner (https://github.com/google/site-kit-wp/issues/8279), however the `bottom: -5px` property doesn't work for that as it leaves a 5px gap at the top. I've taken the following approach instead - you might want to use it here too if it fits the bill:

```suggestion
		display: flex;
		height: 100%;
```

---

Thank you @techanvil I will make use of this.